server/models/Course.js

```diff
@@ -0,0 +1,16 @@
+import mongoose from "mongoose";
+
+const courseSchema = new mongoose.Schema({
+  courseName: { type: String, required: [true, 'courseName cannot be empty'] },
```

Comment:
rename `courseName` to `title`
rename `courseCode` to `code`
rename `credit` to `credits`